docker/images/gatsby/gatsby-init

```diff
@@ -1,27 +1,60 @@
 #!/bin/sh
 set -e
```

Comment:
Please consider making the flags `set -eu` here. That makes things much easier to debug if a variable is undefined due to a typo or some other logical error.

https://github.com/pagely/documentation/blob/master/coding-conventions/bash.md#writing-safe-bug-free-bash